src/hiccup/compiler.clj

```diff
@@ -38,14 +58,19 @@
       value))
 
 (defn- xml-attribute [name value]
-  (str " " (util/as-str name) "=\"" (util/escape-html (render-attr-value value)) "\""))
+  (build-string
+   " "
+   (util/to-str name)
+   "=\""
+   (util/escape-html (render-attr-value value))
+   "\""))
```

Comment:
As a small suggestion, perhaps write it as:

```clojure
  (build-string " " (util/to-str name) "=\""
                (util/escape-html (render-attr-value value)) "\""))
```

That way it uses up less vertical screen real estate.

Reply:
431e741c7c213b3f459c2bb48935566e37ce7ba8